docs/hpc/advanced/profiling/nsight_compute.md

```diff
@@ -0,0 +1,44 @@
+# Nsight Compute
+
+[Nvidia Nsight Compute](https://developer.nvidia.com/nsight-compute) is an interactive kernel profiler for Cuda applications. Once you are satisfied with how your code is interacting with Cpu using [Nsight Systems](./nsight_systems.md), you can now turn to go deeper to see how your kernels are actually using the Gpu.
```

Comment:
CUDA, Cpu and Gpu probably need capitalising.

Reply:
Corrected in 1b89b8fdac7d75afc158fe5655619fa0fd48357c